package.json

```diff
@@ -59,6 +59,7 @@
     "fluxible": "^1.0.3",
     "fluxible-action-utils": "^0.2.4",
     "fluxible-addons-react": "^0.2.0",
+    "gulp-modernizr": "0.0.0",
```

Comment:
I think we should require the latest current version: `gulp-modernizr": "1.0.0-alpha"`.